src/index.ts

```diff
@@ -27,17 +27,27 @@ export class ContentfulLivePreview {
           ContentfulLivePreview.liveUpdates?.receiveMessage(event.data);
         });
 
+        pollUrlChanges(window.location.href, ContentfulLivePreview.handleUrlChange);
+
         sendMessageToEditor({
           action: 'IFRAME_CONNECTED',
           connected: true,
           tags: document.querySelectorAll(`[${TagAttributes.ENTRY_ID}]`).length,
+          url: window.location.href,
```

Comment:
That should be automatically added by `sendMessageToEditor`  as `location` :)

Reply:
Probably also for https://github.com/contentful/live-preview/pull/68/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R47